lib/packagekit-glib2/pk-package-ids.h

```diff
@@ -36,6 +36,7 @@ G_BEGIN_DECLS
  * '|': used as the filename separator
  * '~': conary
  * '@': conary
+ * ',': may be a part of version string in some OSes
```

Comment:
I think we can delete `PK_PACKAGE_IDS_DELIM` entirely now? And maybe move that documentation to the function writing it, expressing why comma is preferred, and that it will be escaped by GKeyFile in places where it is part of the version string.

Reply:
Hmm, the annoying thing is that we can't easily change the delimiter as it is part of the API - I would really like to avoid it. Fortunately, I found zero users of it, but I still wonder if we could just use the same (un)escaping that GKeyFile uses in the helper routines, to solve this problem forever.

Reply:
Just copy it, it should just be string-escaping, right? I can't imagine that that would be complex at all, you can probably write it quickly with a GString and going through input data char-by-char in one go.